prac_06/guitar.py

```diff
@@ -0,0 +1,21 @@
+"""
+Ashton Jack Stewart
+Practical 06 - guitar
+started: 12am
```

Comment:
You needed to guess how long it would take you to complete and how long it actually took you.